### DISCUSSION TITLE
Install of Web2PrintToolsBundle

### DISCUSSION THREAD

**Discussion by chili-666** [pimcore/pimcore#15975] [Q&A / Support] [answered]
Hey there,

I tried to install the Web2PrintToolsBundle into a skeleton pimcore install. pimcore:bundle:list lists the bundle as installed and enabled. I also added Web2PrintToolsBundle\Web2PrintToolsBundle::class => ['all' => true], to my config/bundles.php. Pimcore works without a hitch, but I don't see the menu for Web2Print in my backend.

What am I doing wrong?

P.S. I also ran composer require pimcore/admin-ui-classic-bundle

**Comment by blankse** [comment, marked as answer]
@chili-666 You need to enable/install the Web2PrintBundle https://github.com/pimcore/web-to-print-bundle
The Web2PrintToolsBundle adds additional features for web2print https://github.com/pimcore/web2print-tools

**Reply by chili-666** [reply]
Oh.... I really should get my eyes checkd. I did not see that.... Tanks a lot!
